components/IngredientSelector/CategoryCard.tsx

```diff
@@ -1,18 +1,61 @@
 import { GenericCard } from "@/components/shared/molecules/GenericCard";
+import { BeefIcon } from "@/components/shared/atoms/icons/BeefIcon";
+import { FishIcon } from "@/components/shared/atoms/icons/FishIcon";
+import { EggIcon } from "@/components/shared/atoms/icons/EggIcon";
+import { BroccoliIcon } from "@/components/shared/atoms/icons/BroccoliIcon";
+import { CarrotIcon } from "@/components/shared/atoms/icons/CarrotIcon";
+import { EggplantIcon } from "@/components/shared/atoms/icons/EggplantIcon";
+import { FlourIcon } from "@/components/shared/atoms/icons/FlourIcon";
+import { CerealsIcon } from "@/components/shared/atoms/icons/CerealsIcon";
+import { OliveOil } from "@/components/shared/atoms/icons/OliveOil";
+import { Butter } from "@/components/shared/atoms/icons/Butter";
+import { WineBottle } from "@/components/shared/atoms/icons/WineBottle";
+import { Milk } from "@/components/shared/atoms/icons/Milk";
+import { SaltAndPaper } from "@/components/shared/atoms/icons/SaltAndPaper";
+import { SaladIcon } from "@/components/shared/atoms/icons/SaladIcon";
+import { LeafIcon } from "@/components/shared/atoms/icons/LeafIcon";
 
 interface CategoryCardProps {
   title: string | undefined;
   onClick(): void;
+  categoryId:
+    | "meat-fish-eggs"
+    | "vegetables"
+    | "potatoes-legumes-cereals"
+    | "fats"
+    | "liquids"
+    | "spices-condiments"
+    | "fresh-herbs";
 }
 
-export function CategoryCard({ title, onClick }: CategoryCardProps) {
+export function CategoryCard({
+  title,
+  onClick,
+  categoryId,
+}: CategoryCardProps) {
+  console.log(categoryId);
+  const IconsMap = {
+    "meat-fish-eggs": [
+      <BeefIcon key={"beef"} />,
```

Comment:
Second dumb question: what is the purpose of the key={"beef"}

Reply:
We need to have a key property for each element or the array to avoid bugs so I did this (but to be honest I'm not sure that it was the best solution).

Reply:
Yes, it's not super usual to see fix array of items. Woudl it work with the following format

```jsx
"meat-fish-eggs": <React.Fragment>
    <BeefIcon />
    <EggIcon />
  </React.Fragment>
```

Reply:
Another option is to define 

const IconsMap = {
 "meat-fish-eggs": ["/static/illustration/eggs.svg", "/static/illustration/beef.svg"],
 ...
}

And in the `<GenericCards />` replace the `svg` by `illustrations`, an array of src 


```diff
{illustrations && (
          <Stack direction={"row"} justifyContent={"center"} gap={1}>
-            {svg}
+            {illustrations.map(src => <img src={src} alt="" key={src} />)}
          </Stack>
        )}
        ```

Reply:
> Another option is to define
> 
> const IconsMap = { "meat-fish-eggs": ["/static/illustration/eggs.svg", "/static/illustration/beef.svg"], ... }
> 
> And in the `<GenericCards />` replace the `svg` by `illustrations`, an array of src
> 
> ```diff
> {illustrations && (
>           <Stack direction={"row"} justifyContent={"center"} gap={1}>
> -            {svg}
> +            {illustrations.map(src => <img src={src} alt="" key={src} />)}
>           </Stack>
>         )}
>         ```
> ```

This would be my preferred option (less code, easier to edit / add SVG files directly). But we can do it later.